vcap/setup.py

```diff
@@ -1,18 +1,23 @@
 #!/usr/bin/env python3
 import os
+from pathlib import Path
 
 from setuptools import setup, find_namespace_packages
 
+# Get package version/metadata
+about = {}
+exec(Path("vcap/version.py").read_text(), about)
```

Comment:
What happens if you just use a normal import?

Reply:
If I use a normal import, i'm importing from the `vcap` package (and __init__.py) which pulls in all the dependencies I am about to install. 

This approach was used in Requests, but without pathlib (since I guess they support old python version)

Reply:
I'm not sure... The only reason I felt comfortable with this approach is because other major repo's do it, and because it was one of the recommended way of solving this problem (the "single version in repository" problem)

Reply:
Thoughts, Bryce? 

In this use, I think it's equivalent to importing a module. It's not arbitrary (it's the package that is being pulled in by the pypi or our git clones), so those are signed/protected and shouldn't be dangerous. It's also not arbitrary in that it's our code, not a string received from an API.

Reply:
well it's arbitrary in the sense that theoretically a malicious actor could replace `vcap/version.py` with a different file;
but yeah it's probably fine